### ISSUE TITLE
DeprecationWarning: '_app_ctx_stack' is deprecated and will be removed in Flask 2.4.

### ISSUE THREAD

**Issue by eladavron** [maxcountryman/flask-seasurf#144]
Getting this warning in my pytests:
```
.venv/lib/python3.11/site-packages/flask_seasurf.py:28: DeprecationWarning: '_app_ctx_stack' is deprecated and will be removed in Flask 2.4.
```

**Comment by alanhamlett** [comment]
Try installing from `main` branch because that was fixed 2 yrs ago in #132 just not released to pypi:

```
pip install -e git+https://github.com/maxcountryman/flask-seasurf.git@f383b482c69e0b0e8064a8eb89305cea3826a7b6#egg=Flask-SeaSurf
```

**Comment by eladavron** [comment]
Thanks, will try.  
Is this project no longer maintainer?  
Is there an alternative/fork?

**Comment by alanhamlett** [comment]
It's maintained and working, just need to remind @maxcountryman to do a pypi release.

**Comment by alanhamlett** [comment]
Just noticed I can publish releases. Released v2.0.0 now.
